NewForce-Cohort-7/tabloidfullstack-thegroundbeetles
# Gr tabloid19

# Description
Ticket # 21: https://trello.com/c/lToUwCfU/21-21-delete-a-comment

The functionality to delete a comment associated with a post has been added. Given a user is logged in the app & in the post detail page of a post,  they should be able to click on the delete comment button.Then a message confirming users wish to delete the comment will be displayed. When the user clicks on the delete button the comment will be removed from the database and the DOM/UI. The user also can click the cancel button if they choose to no longer delete the comment.


# Type of change:
*New feature (non-breaking change which adds functionality)


# Testing Instructions:

- Reference [How to test a teammate's PR](https://gist.github.com/jordan-castelloe/96f76d5bd4cfbd2181084af674c2a1af#when-you-need-to-test-a-teammates-pr)
- git checkout grTabloid19
- If you don't see the most recent version of my files, git pull origin grTabloid19
- Add all seed data by adding a new query and copying the code from the links below
- Use this to fill out tables and columns:
https://github.com/NewForce-Cohort-7/tabloidfullstack-thegroundbeetles/blob/main/SQL/01_Tabloid_Create_DB.sql
- Use this to seed the database:
https://github.com/NewForce-Cohort-7/tabloidfullstack-thegroundbeetles/blob/main/SQL/02_Tabloid_Seed_Data.sql
- To execute your query just press Ctrl+Shift+e inside of your query.
- Run the program by clicking the Green Arrow next to TabloidMVC at the top.
- In the  browser, please go to Login on the nav bar 
- Log in with this email for both the email and password: [rsandwith0@google.com.brx](mailto:rsandwith0@google.com.brx). This should direct you back to the home page.
- Select Post from the navbar
- Click on the title of a post to be directed to the post details page
- Click on Add Comment button
- Enter values in the Subject and Content field then click submit
-Users will be redirected to the post details page
- Click view comments button to see the newly entered comment
- Click on the delete comment button of your comment.
- A message confirming the wish to delete the comment will be displayed
- When the user clicks on the delete button the comment will be removed from the database and the DOM/UI. 
-The user also can click the cancel button if they choose to no longer delete the comment.
- Locate Logout on the Navbar & click this to log out.


# Checklist:

- [ x] - My code follows the style guidelines of this project
- [x ] - I have performed a self-review of my code
- [x ] - I have commented my code, particularly in hard-to-understand areas
- [x ] - My changes generate no new warnings or errors
- [x ] - I have added test instructions that prove that my feature works
